repository/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -88,4 +88,7 @@
     <include file="db/changelog/db.changelog-rdm-8382.xml"/>
     <include file="db/changelog/db.changelog-rdm-8509.xml"/>
     <include file="db/changelog/db.changelog-rdm-8692.xml"/>
+    <include file="db/changelog/db.changelog-rdm-2676.xml"/>
+    <include file="db/changelog/db.changelog-rdm-8869.xml"/>
```

Comment:
this looks like its was added by a merge from develop - not part of RDM-9024

Reply:
Thanks @RebeccaBaker, log updated